xrpl/asyncio/wallet/wallet_generation.py

```diff
@@ -135,7 +135,10 @@ async def _check_wallet_balance(address: str, client: Client) -> int:
 
 async def _request_funding(url: str, address: str) -> None:
```

Comment:
I think an optional parameter of usecase would be helpful.  That way we can say 'unit test' or 'integration test'.

Reply:
added